client/datalake/tests/conftest.py

```diff
@@ -82,6 +77,10 @@ def random_abs_dir():
 
 @pytest.fixture
 def random_metadata():
+    return generate_random_metadata()
+
+
+def generate_random_metadata():
```

Comment:
Hm. Was there a problem with just invoking `random_metadata`? I guess it's semantically sensible to add `generate` to the front of this. But I'd rather just stick to the existing `random_metadata` instead of exposing two functions that essentially do the same thing.

Reply:
Got it.